Client/Frontend/Brave Today/FeedItemView.swift

```diff
@@ -0,0 +1,474 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+
+/// Defines the view for displaying a specific feed item given a specific layout
+///
+/// Use the lazy variables to access and display the correct data for the
+/// correct layout.
+class FeedItemView: UIView {
+    /// The layout of the feed item
+    private let layout: Layout
+    /// Create a feed item view using a given layout. Each layout may use
+    /// any combination of the thumbnail, title, item date, and branding
+    init(layout: Layout) {
+        self.layout = layout
+        super.init(frame: .zero)
+        let contentView = view(for: .stack(layout.root))
+        addSubview(contentView)
+        contentView.snp.makeConstraints {
+            $0.edges.equalToSuperview()
+        }
+        isAccessibilityElement = true
+    }
+    /// The feed thumbnail image view. By default thumbnails aspect scale to
+    /// fill the available space
+    lazy var thumbnailImageView = UIImageView().then {
+        $0.contentMode = .scaleAspectFill
+        $0.backgroundColor = UIColor(white: 1.0, alpha: 0.1)
+        $0.clipsToBounds = true
+    }
+    /// The feed title label
+    var titleLabel = UILabel().then {
```

Comment:
@kylehickinson - AFAIK, UILabels() render html/js content as plain-text so there is no possibility of an XSS attack. I only see UILabels being used to display text, are there any other UI elements that are being used that might result in such issues?

Reply:
Not that I know of, we don't have anything special UI-wise for Brave Today